draft-thomson-escape-report.md

```diff
@@ -280,14 +294,19 @@ stated goals of Web Packaging is to enable sharing of content offline.
 
 Several participants reported that in areas where Internet access is expensive,
 slow, or intermittent, the use of direct peer-to-peer file exchange (e.g.,
-"saving a Web site and sharing it on a USB stick") is commonplace. Most Web
-browsers already have some affordances for this, but these are recognized as in
-need of improvements.
+"saving a Web site and sharing it on a USB stick or direct file transfer") is
```

Comment:
I don't think this really helps. "Direct" from where?

Reply:
It's usually via WiFi Direct or Bluetooth, but this isn't a huge deal.